frontend/src/Editor/Components/FilePicker.jsx

```diff
@@ -105,6 +107,19 @@ export const FilePicker = ({
   const [accepted, setAccepted] = React.useState(false);
   const [showSelectdFiles, setShowSelectedFiles] = React.useState(false);
   const [selectedFiles, setSelectedFiles] = React.useState([]);
+  //* custom validator
+  function validateFileExists(file) {
+    const isValid = selectedFiles.some((selectedFile) => selectedFile.name === file.name);
```

Comment:
@arpitnath Shouldn't this be based on the same full file path instead of same file name? Because different files can have same name, right?

Reply:
Yeah, makes sense. I think we need to have the selected files' **paths** and compare them instead of their names.